components/product-list.html

```diff
@@ -0,0 +1,64 @@
+<div class="bg-white">
+    <div class="  bg-gray-100 max-w-4xl mx-auto py-16 px-4 sm:py-24 sm:px-6 lg:max-w-7xl lg:px-8">
+      <h2 class="text-2xl text-gray-900 font-extrabold tracking-tight ">Product-List</h2>
```

Comment:
```suggestion
      <h2 class="text-2xl text-gray-900 font-extrabold tracking-tight ">Product List</h2>
```
There is  no need to add a dash `-` between the words, if your could you can remove them all. 🙂

Reply:
Ok .I will change it